### DISCUSSION TITLE
OpenAI API LiteLLM proxy server

### DISCUSSION THREAD

**Discussion by mkagit** [jakobdylanc/llmcord.py#68] [General]
Hi!
Since I'm already running a LiteLLM proxy server for another project I'm wondering if I could use it also for the llmcord. I use it with virtual keys for tracking usage, set budget, guardrails and to attach a group of models to a team virtual key. I manage all from the Admin UI. 
Is it possible to set the baseUrl as the LiteLLM OpenAI API compatible endpoint? Would be like this: http://myserver.com:4000/v1/chat/completions

**Comment by jakobdylanc** [comment]
Yes this should be possible. Don't include the `/chat/completions` part. Just add this new entry to `providers` in your config.yaml:

```yaml
providers:
  litellm:
    base_url: http://myserver.com:4000/v1
```

And then set `model` to `litellm/<model name>`.
